### PR TITLE
feat/users add user configmap, template to its configmap and some updates

### PR DESCRIPTION
**Pull request by weslleycamilo**
I've updated it so we can create a new EKS cluster and add some users after its creation.

[AWS documentation about new users](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html)

P.S. to obtain EKS access there is a need to edit the auth-aws configmap so you can update it with the new users with their AWS ARNs 
Once we don't have a functional pattern from our user vars I had to create a new one with a dot(.) .

### CODE REVIEW

File: tasks/bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+- name: Create ConfigMap yml from jinja template
+  template:
+    src: users_configMap.j2
+    dest: /tmp/users_configMap.yml
+
+- name: Read definition file and update auth-aws ConfigMap
+  k8s:
+     state: present
+     kubeconfig: /home/weslley/.kube/config
```
Comment (weslleycamilo) [inline tasks/bootstrap.yaml]: I've to find a better solution for this. I don't know why I've to specify it once its is already the default path.

I know that it breaks if I don't specify, its breaks even if I specify what they suggest in the documentation:  https://docs.ansible.com/ansible/latest/modules/k8s_module.html

File: defaults/main.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ eks_deploy: false
 
 # EKS cluster name
 eks_cluster_name: dev
+eks_aws_account_id: 455897974533
```
Comment (weslleycamilo) [inline defaults/main.yml]: It's a random number once its a public repo.

File: tasks/main.yml [hunk 3]
```diff
@@ -7,3 +7,6 @@
 - include: worker.yaml
 
 - include: ingress.yaml
+
+- include: bootstrap.yaml
+  tags: bootstrap
```
Comment (weslleycamilo) [inline tasks/main.yml:10]: I've create its tag so we can use it even after EKS cluster creation.

File: templates/users_configMap.j2 [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  mapRoles: |
+    - rolearn: {{ eks_iam_role_arn_worker }}
+      username: system:node: {{'{{'}}EC2PrivateDNSName{{'}}'}}
+      groups:
+       - system:bootstrappers
+       - system:nodes
+  mapUsers: |
+{% if eks_iam_users is defined %}
+{% for user in eks_iam_users_arn %}
+    - userarn: arn:aws:iam::{{ eks_aws_account_id }}:user/{{ user }}
```
Comment (weslleycamilo) [inline templates/users_configMap.j2:12]: well I believe it is good way to insert its aws account id here so the user/dev won't need to change the template.

do you all agree ? is there any better way ? I am open to suggestions.

**Review by alecventura — APPROVED** [review]
LGTM

**Pull request merged**